src/stateToHTML.js

```diff
@@ -157,19 +156,19 @@ class MarkupGenerator {
   inlineStyles: StyleMap;
   styleOrder: Array<string>;
 
-  constructor(contentState: ContentState, options: ?Options) {
-    if (options == null) {
-      options = {};
-    }
-    this.contentState = contentState;
-    this.options = options;
-    let [inlineStyles, styleOrder] = combineOrderedStyles(
-      options.inlineStyles,
-      [DEFAULT_STYLE_MAP, DEFAULT_STYLE_ORDER],
-    );
-    this.inlineStyles = inlineStyles;
-    this.styleOrder = styleOrder;
-  }
+  constructor(contentState: ContentState, options: ?Options) {
+    if (options == null) {
+      options = {};
+    }
+    this.contentState = contentState;
+    this.options = options;
+    let [inlineStyles, styleOrder] = combineOrderedStyles(
+      options.inlineStyles,
+      [DEFAULT_STYLE_MAP, DEFAULT_STYLE_ORDER],
+    );
+    this.inlineStyles = inlineStyles;
+    this.styleOrder = styleOrder;
+  }
```

Comment:
No change really, must be some spacing or something changed just be opening and saving the file in my editor. I couldn't manage to unstage the hunk either using SourceTree.